src/pkg/selectors/sharepoint_test.go

```diff
@@ -352,18 +355,58 @@ func (suite *SharePointSelectorSuite) TestSharePointRestore_Reduce() {
 
 func (suite *SharePointSelectorSuite) TestSharePointCategory_PathValues() {
 	t := suite.T()
-
 	pathBuilder := path.Builder{}.Append("dir1", "dir2", "item")
-	itemPath, err := pathBuilder.ToDataLayerSharePointPath("tenant", "site", path.LibrariesCategory, true)
-	require.NoError(t, err)
-
-	expected := map[categorizer]string{
-		SharePointSite:        "site",
-		SharePointLibrary:     "dir1/dir2",
-		SharePointLibraryItem: "item",
+	ten := "tenant"
+	site := "site"
+	table := []struct {
+		name     string
+		pb       func() (path.Path, error)
+		sc       sharePointCategory
+		expected map[categorizer]string
+	}{
+		{
+			name: "SharePoint Libraries",
+			sc:   SharePointLibraryItem,
+			expected: map[categorizer]string{
+				SharePointSite:        site,
+				SharePointLibrary:     "dir1/dir2",
+				SharePointLibraryItem: "item",
+			},
+			pb: func() (path.Path, error) {
+				return pathBuilder.ToDataLayerSharePointPath(
+					ten,
+					site,
+					path.LibrariesCategory,
+					true,
+				)
+			},
```

Comment:
This is a little hefty for the test.  The only variable changing between them is the category type, which you can get from `test.sc.PathType()`.  That way the pathBuilder call can stay inside the subtest.

Reply:
Resolved within [32717af](https://github.com/alcionai/corso/pull/1786/commits/32717af4e57329266e1047d0cb793e35d454c92f)